addons/website_event/static/src/js/website_event.js

```diff
@@ -12,13 +11,18 @@ var EventRegistrationForm = publicWidget.Widget.extend({
      */
     start: function () {
         var self = this;
+        var post = {};
+        $('#registration_form select').each(function () {
+            post[$(this).attr('name')] = $(this).val();
+        });
+        const noTicketsOrdered = Object.values(post).map((value) => parseInt(value)).every(value => value === 0);
         var res = this._super.apply(this.arguments).then(function () {
             $('#registration_form .a-submit')
                 .off('click')
                 .click(function (ev) {
                     self.on_click(ev);
                 })
-                .prop('disabled', false);
+                .prop('disabled', (noTicketsOrdered)?true:false);
```

Comment:
`noTicketsOrdered` is already a boolean value :slightly_smiling_face: 
So you could just do:
```js
.prop('disabled', noTicketsOrdered);
```